hazelcast-jet-distribution/src/root/config/log4j.properties

```diff
@@ -13,14 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+log4j.rootLogger=INFO, stdout, rollingFile
```

Comment:
Logging to `stdout` in daemon mode is bad. We should address this in future work.